packages/block-editor/src/components/iframe/index.js

```diff
@@ -264,6 +271,8 @@ function Iframe(
 				{ ...props }
 				ref={ useMergeRefs( [ ref, setRef ] ) }
 				tabIndex={ tabIndex }
+				// Correct doctype is required to enable rendering in standards mode
+				srcDoc="<!doctype html><html><head>yyy</head><body>xxx</body></html>"
```

Comment:
Here you only need to create the `<html>` element, with no content. The `createPortal` call will populate the `.documentElement` (which points to the `html` element) with `head` and `body`. As is the custom, the root element where React UI is mounted should be empty.

Reply:
Good catch - removing - and left some debug noise on top.